modules/ROOT/partials/misc/admon-ai-proxy.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: For the sake of demonstration, the following examples show how to use the authentication credentials with the API within the client side integration. This is not recommended for production purposes. It is recommended to only access the API with a xref:ai-proxy.adoc[proxy server] or by implementing a server-side integration to prevent unauthorized access to the API.
```

Comment:
```suggestion
IMPORTANT: The following examples are intended to show how to use the authentication credentials with the API within the client side integration. This is **not recommended** for production purposes. It is recommended to only access the API with a xref:ai-proxy.adoc[proxy server] or by implementing a server-side integration to prevent unauthorized access to the API.
```